package.json

```diff
@@ -270,12 +270,12 @@
   },
   "scripts": {
     "vscode:prepublish": "npm run compile",
-    "package": "vsce package",
-    "compile": "tsc -b",
-    "watch": "tsc -b -w",
-    "lint": "eslint src/**/*.ts",
-    "test-compile": "tsc -p ./",
-    "test": "npm run test-compile && node ./out/test/runTest.js"
+    "compile": "tsc -p ./",
+    "watch": "tsc -watch -p ./",
```

Comment:
I like that these long flags avoid the need to read manual in most cases, is it worth also replacing `-p` with `-project`?

btw. `tsc --help` doesn't explicitly say that these long single-hyphen options are supported, but I suppose they are. 🤷🏻‍♂️

Reply:
Conventions? Parameter specifications rules? Who needs them? 😭 

I'm following conventions I've seen in other projects here, but if you want more concrete reasoning I'll look into it.